evcouplings/couplings/protocol.py

```diff
@@ -561,7 +644,228 @@ def complex(**kwargs):
             chain=chain_mapping
         )
     }
-    
+
+    # save just the inter protein ECs
+    ## TODO: eventually have this accomplished by _postprocess_inference
+    ## right now avoiding a second call with a different ec_filter
+    ecs = pd.read_csv(outcfg["ec_file"])
+    outcfg["inter_ec_file"] = prefix + "_CouplingScores_inter.csv"
+    inter_ecs = ecs.query("segment_i != segment_j")
+    inter_ecs.to_csv(outcfg["inter_ec_file"], index=False)
+
+    # dump output config to YAML file for debugging/logging
+    write_config_file(prefix + ".couplings_complex.outcfg", outcfg)
+
+    # TODO: make the following complex-ready
+
+    # EVzoom:
+    #
+    # 1) at the moment, EVzoom will use numbering before remapping
+    # we should eventually get this to a point where segments + residue
+    # index are displayed on EVzoom
+    #
+    # 2) note that this will currently use the default mixture model
+    # selection for determining the EC cutoff, rather than the selection
+    # used for the EC table above
+
+    return outcfg
+
+def complex_mean_field(**kwargs):
```

Comment:
This carries way too much redundancy with mean_field() (comment I made originally with Kelly's PR containing this piece of code)

Please refactor in same way that standard() and complex() use infer_plmc(), i.e. there should be a similar infer_mean_field() function that contains all the shared code for mean-field inference, and only the last bits that are specific to monomer vs complex should be in the functions mean_field() and complex_mean_field()